docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -5,8 +5,7 @@ type: Getting Started
 
 ## Overview
 
-This Getting Started guide instructs developers to quickly deploy Kyma locally on a Mac, Linux, or Windows. Kyma installs locally using a proprietary installer based on a Kubernetes operator.
-The document provides the prerequisites, the instructions on how to install Kyma locally and verify the deployment, and the troubleshooting tips.
+This Getting Started guide shows developers how to to quickly deploy Kyma locally on a Mac, Linux, or Windows. Kyma installs locally using a proprietary installer based on a Kubernetes operator. The document provides prerequisites, instructions on how to install Kyma locally and verify the deployment, as well as the troubleshooting tips.
```

Comment:
Kubernetes operator - you can link to a website that explains the concept: https://coreos.com/operators/